js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts

```diff
@@ -301,6 +301,12 @@ export const PROMPT_TEMPLATE_VARIABLES =
 export const PROMPT_TEMPLATE_TEMPLATE =
   `${PROMPT_TEMPLATE_PREFIX}.template` as const;
 
+/**
+ * A prompt template version
+ */
+export const PROMPT_TEMPLATE_VERSION =
```

Comment:
not sure how we want to handle these we might be behind on a few of them so can accumulate here then do a patch and rebase before merging the feture branch

Reply:
can you cherry pick this and PR main? That's the safest option. In theory this should work. We can try it if you want.